modules/KoreBuild.Tasks/DisableSkipStrongName.cs

```diff
@@ -0,0 +1,70 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using KoreBuild.Tasks.SkipStrongNames;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Microsoft.Win32;
+
+namespace KoreBuild.Tasks
+{
+    public class DisableSkipStrongName : Task
+    {
+        [Required]
+        public string XmlFile { get; set; }
```

Comment:
I'll leave it to you - you could pass this in as an ItemGroup. Not sure it's worth the effort since the current code works.